### PR TITLE
About

### PR DESCRIPTION
**Pull request by samjett247**
- Handled routing to/from the about and getinvolved pages
- Modified the hamburger menu for phone view
- Modified yaxes of timeseries so they dont adjust any longer

@schuermannator Lets get this merged then I'd love your help with the getinvolved page

### CODE REVIEW

File: src/components/GetInvolved.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import '../App.css';
+
+
+class GetInvolved extends React.Component {
+  constructor(props) {
```
Comment (schuermannator) [inline src/components/GetInvolved.js]: useless constructor?

File: src/components/GetInvolved.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    super(props);
+  }
+  componentDidMount() {
```
Comment (schuermannator) [inline src/components/GetInvolved.js]: delete unless we need lifecycle methods

File: src/App.css [hunk 3]
```diff
@@ -1,13 +1,23 @@
+/* This is for sticky navbar */
+body { padding-top: 100px; }
```
Comment (schuermannator) [inline src/App.css]: formatting, I can look into prettier sort of thing for CSS

File: src/components/DisclaimerAlert.js [hunk 4]
```diff
@@ -30,8 +41,12 @@ class DisclaimerAlert extends React.Component {
         closeOnClickOutside: false,
       });
     }
-    this.props.alertUser();
   }
+  componentDidUpdate() {
+    if (!this.props.user_alerted) {
+
```
Comment (schuermannator) [inline src/components/DisclaimerAlert.js]: TODO

**Comment by schuermannator** [comment]
did the make pretty

**Comment by samjett247** [comment]
Fixed everything you mentioned. Let's merge her in when ready.

File: src/components/About.js [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+            prior to enrollment, students typically turn to third-party
+            resources, such as{' '}
+            <a href="https://www.ratemyprofessors.com">Rate My Professor</a>.
+            These external services lack adequate reviews for many professors
+            while representing only the students who took the time to contribute
+            a response - typically, these are the students who either loved or
+            hated a course or instructor. With these external tools as the only
+            option, students at OU are forced to either: enroll in courses with
+            inadequate prior knowledge of the course quality, or base their
+            enrollment decisions on highly-biased evaluations. We started this
```
Comment (schuermannator) [inline src/components/About.js]: Last sentence: "We created this application to provide..."

File: src/components/About.js [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+            project to provide a superior alternative to these unappealing
+            options.
+          </p>
+          <h3>Project Goal</h3>
+          <p>
+            We wanted to provide a visualization of course reviews that are{' '}
```
Comment (schuermannator) [inline src/components/About.js]: change wanted -> want

File: src/components/About.js [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+            <i>validated</i>, <i>extensive</i>, and <i>representative</i> of the
+            enrolled students. Fortunately, verified student evaluations of
+            teaching are administered through OU at <i>eval.ou.edu</i> at the
```
Comment (schuermannator) [inline src/components/About.js]: student evaluations of courses? idk I think student evaluations of _teaching_ sounds weird.

File: src/components/About.js [hunk 8]
```diff
@@ -0,0 +1,149 @@
...
+            end of each term. These evaluations are stored by the University and
+            used by departments to analyze their programs and course-instructor
+            matching. We believe students at OU should have a right to this
+            information, and have built this website to offer functional,
+            actionable views of the OU course evaluation data. Wehope that
```
Comment (schuermannator) [inline src/components/About.js]: Wehope typo

File: src/components/About.js [hunk 9]
```diff
@@ -0,0 +1,149 @@
...
+            students will use these evaluations as they plan their courses at
+            OU, and have received unanimous approval from the{' '}
+            <span id="SETresearch">OU</span>{' '}
+            <a href="http://www.ou.edu/content/dam/sga/common/USG/Minutes/Session%20102/Congress%20Minutes%2004-23-19.pdf#page=5">
+              Undergraduate Student Congress
+            </a>{' '}
+            attesting to the student interest in and support for this project.
```
Comment (schuermannator) [inline src/components/About.js]: 💯 nice sentence

File: src/components/About.js [hunk 10]
```diff
@@ -0,0 +1,149 @@
...
+            <a href="https://it.wisc.edu/services/aefis/">AEFIS</a> at the
+            University of Wisconsin, and{' '}
+            <a href="https://accessuh.uh.edu/login.php">AccessUH</a> at the
+            University of Houston allow students to access evaluations.
+            Currently, the OU internal reviews are{' '}
+            <a href="https://www.ou.edu/provost/course-evaluation-data">
+              available publicly
+            </a>
+            , but are stored in vast collections of PDF files and are extremely
+            difficult to access or draw insights from. We started this project
```
Comment (schuermannator) [inline src/components/About.js]: started -> 'created'

File: src/components/About.js (hunk 6 above)
Comment (samjett247) [reply src/components/About.js]: Changed "We want" to "Our goal is"

File: src/components/About.js (hunk 7 above)
Comment (samjett247) [reply src/components/About.js]: I dig. Changed to "courses and instructors"

File: src/components/About.js (hunk 8 above)
Comment (samjett247) [reply src/components/About.js]: Done

File: src/components/About.js (hunk 9 above)
Comment (samjett247) [reply src/components/About.js]: TY

**Pull request merged**